src/app/containers/AudioVideo/Canonical.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
This file reports 25% coverage which is probably where your codeclimate failure lies

Reply:
(fixed)